### ISSUE TITLE
double-right click osx doesn't seem to work

### ISSUE THREAD

**Issue by saberworks** [marklieberman/foxygestures#200]
The double-right click on OSX doesn't seem to be working for me.  I have OSX and FF 57.04 and the latest version of this addon from addons.mozilla.org.  When the double-click context option is enabled (which is the default, but I also tried disabling, and then re-enabling, and also tried restarting my browser, same issue), right-click to initiate a gesture always brings up the context menu.  My understanding was that if this option is checked, the context menu should not pop up unless I double-right-click, and that mouse gestures should work "normally" otherwise (single right-click and drag).  Is there some other preference or option that could be interfering with this?  I am using a regular mouse attached to a mac laptop running 10.12.6 (sierra).  If there's any other information I can provide, please let me know.

**Comment by saberworks** [comment]
I figured out what was causing this.  There's an about:config setting called `dom.event.contextmenu.enabled` that I had set to `false`  When flipped to true, the double-right-click setting works correctly.

I wish there was a way to tell firefox to allow addons to mess with the context menu but not scripts provided by web sites.  The problem is that web sites abuse the privilege.  They disable things like copy/paste or "save as" or they disable the menu completely (as some sort of ill-thought-out copy-protection scheme).
